src/libcore/num/mod.rs

```diff
@@ -2504,6 +2504,30 @@ impl fmt::Display for TryFromIntError {
     }
 }
 
+#[unstable(feature = "try_from", issue = "33417")]
+impl TryFrom<bool> for bool {
+    type Error = ();
```

Comment:
Actually this one should be covered by the (not yet existing) blanket impl `impl<T, U> TryFrom<T> for U where U: From<T>`.